test/controllers/businessmatching/SummaryControllerSpec.scala

```diff
@@ -42,7 +45,7 @@ class SummaryControllerSpec extends AmlsSpec with BusinessMatchingGenerator {
     val mockBusinessMatchingService = mock[BusinessMatchingService]
 
     val controller = new SummaryController (
-      dataCache = mockCacheConnector,
+      dataCacheConnector = mockCacheConnector,
```

Comment:
Do we need this line given we call mockCacheConnector directly?

Reply:
Removed